### DISCUSSION TITLE
Basket / Catalog Price Rules - Overlap doesn' t use the biger discoun't

### DISCUSSION THREAD

**Discussion by juliomgm** [PrestaShop/PrestaShop#33438] [Ideas]
### Prerequisites

- [x] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

If we create 2 Basket / Catalog Price Rules that overlap in some products, prestashop picks the discount of the first one it finds (i think). But by law in my country (PT) it must be the higher discount that must be used. I looked for this config and also online but can't find anytime about this.

### Expected behavior

If  2 Basket / Catalog Price Rules overlap prestashop must use the one with higher discount. 

### Steps to reproduce

Admin / Backoffice

..../admin/index.php?controller=AdminSpecificPriceRule

### PrestaShop version(s) where the bug happened

1.7.7.3

### PHP version(s) where the bug happened

7.4

### Your company or customer's name goes here (if applicable).

Servidor Informatica

**Comment by kpodemski** [comment]
Hello @juliomgm 

What you are writing about is more like a feature request than a bug. I'll convert this to the Discussions, maybe some other community users from Portugal encountered similar issues.

**Comment by juliomgm** [comment]
Can anyone point me to where is the sql for the Catalog Price Rules so i could change the order by reduction_type, reduction
